packages/client/src/runtime/RequestHandler.ts

```diff
@@ -149,7 +151,7 @@ export class RequestHandler {
         const result = await engineHook(
           {
             document,
-            runInTransaction,
+            runInTransaction: Boolean(transaction),
```

Comment:
This is a part of public middleware API, so here `runInTransaction` stays, it just gets derived from `transaction` field.

Reply:
I don't really know if we made this api public, by looking at the generated types

Reply:
Ah, i see, this is a different type of middleware